solutions/USACO dec2020-cowntagion.mdx

```diff
@@ -0,0 +1,121 @@
+---
+id: usaco-1062
+source: USACO Silver 2020 February
+title: Cowntagion
+author: Kevin Sheng
+---
+
+[Official Editorial](http://www.usaco.org/current/data/sol_prob1_silver_dec20.html)
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+/**
+ * 2020 dec silver
+ * 4
+ * 1 2
+ * 1 3
+ * 1 4 should output 5
+ */
+public class Cowntagion {
+    public static void main(String[] args) throws IOException {
+        BufferedReader read = new BufferedReader(new InputStreamReader(System.in));
+        int farmNum  = Integer.parseInt(read.readLine());
+        ArrayList<Integer>[] neighbors = new ArrayList[farmNum];
+        for (int f = 0; f < farmNum; f++) {
+            neighbors[f] = new ArrayList<>();
+        }
+        for (int i = 0; i < farmNum - 1; i++) {  // the thing is a tree if you didn't know alr
+            StringTokenizer path = new StringTokenizer(read.readLine());
+            int farm1 = Integer.parseInt(path.nextToken()) - 1;
+            int farm2 = Integer.parseInt(path.nextToken()) - 1;
+            neighbors[farm1].add(farm2);
+            neighbors[farm2].add(farm1);
+        }
+
+        ArrayList<Integer>[] spreadTo = new ArrayList[farmNum];
+        for (int f = 0; f < farmNum; f++) {
+            spreadTo[f] = new ArrayList<>();
```

Comment:
Why do you need a spread to when you already stored the edges in the adjacency list?